run_analysis.py

```diff
@@ -95,69 +145,79 @@ def analysis(config):  # {{{
     # analysis can only be imported after the right MPL renderer is selected
 
     # GENERATE OCEAN DIAGNOSTICS
-    if config.getboolean('ohc_timeseries', 'generate'):
+    if checkGenerate(config, analysisName='timeSeriesOHC', mpasCore='ocean',
+                     analysisCategory='timeSeries'):
         print ""
         print "Plotting OHC time series..."
         from mpas_analysis.ocean.ohc_timeseries import ohc_timeseries
         ohc_timeseries(config, streamMap=oceanStreamMap,
                        variableMap=oceanVariableMap)
 
-    if config.getboolean('sst_timeseries', 'generate'):
+    if checkGenerate(config, analysisName='timeSeriesSST', mpasCore='ocean',
+                     analysisCategory='timeSeries'):
         print ""
         print "Plotting SST time series..."
         from mpas_analysis.ocean.sst_timeseries import sst_timeseries
         sst_timeseries(config, streamMap=oceanStreamMap,
                        variableMap=oceanVariableMap)
 
-    if config.getboolean('nino34_timeseries', 'generate'):
+    if checkGenerate(config, analysisName='timeSeriesNino34',
+                     mpasCore='ocean', analysisCategory='timeSeries'):
         print ""
         print "Plotting Nino3.4 time series..."
```

Comment:
@xylar: I would comment out the print statements that are associated with analysis that is not yet available, because they may be cause of confusion.